testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,83 @@
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['build', '-t', 'gcr.io/${PROJECT_ID}/${_SERVICE}:${COMMIT_SHA}', '$_SAMPLE']
```

Comment:
But after thinking more clearly about this... the argument for build should probably be `.`. The use of the sample directory name in Cloud Build came from running the cloud build command a directory up. Are we doing that in this repo given we plan to keep the CI config inside the sample directories?

Reply:
Cloud Build loads the repo into `workspace/` when triggered, therefore we still need to direct cloud build to the code. I can replace with `.` and add the arg `dir: $_SAMPLE`, if that is more desirable.

Reply:
I see. I think `dir: $_SAMPLE` may be better because we'll have a discrete property we can document. The conundrum as I see it is that I had this notion of local dev flow being able to operate with zero context outside the sample directory.

In that case, the instructions for manual trigger might look like:

```sh
cd $_SAMPLE
gcloud builds submit --substitutions '_SAMPLE=.,COMMIT_SHA=manual'
```

I'm comfortable with that, if it makes sense?

Reply:
I like that. I've added it to the README as well.